src/lib/stores/networksStore.ts

```diff
@@ -0,0 +1,54 @@
+import { writable, type Writable } from 'svelte/store';
+
+/**
+ * Represents the available networks.
+ */
+export enum Network {
+  NONE = 'NONE',
+  TESTNET = 'TESTNET',
+  MAINNET = 'MAINNET',
+  LOCALHOST = 'LOCALHOST',
+  CUSTOM = 'CUSTOM',
+}
+
+/**
+ * Represents information about a network.
+ */
+export interface NetworkInfo {
+  name: string | undefined;
+  endpoint: URL | undefined;
+  genesisHash: string | undefined;
+}
+
+/**
+ * Mapping of network types to network information.
+ */
+export const networkToInfo: Record<Network, NetworkInfo> = {
+  NONE: { name: undefined, endpoint: undefined, genesisHash: undefined },
```

Comment:
do we need to define or can we just put an empty object?

Reply:
The key matches the `Network` enum values.

Reply:
Yes we need to define the key NONE. I was just talking about the object that we're assigning it. To assign it an empty array I think will work the same.